client-js/app/elements/labrad-registry.html

```diff
@@ -25,8 +24,8 @@
       height: 100%;
     }
 
-    .iron-selected {
-      background-color: #ddd;
+    tbody tr:nth-child(1n).iron-selected {
+      background-color: #CCCCCC;
```

Comment:
Again the newlines between css selectors. This is probably my fault from way back.